mobile_app/.packages

```diff
@@ -24,17 +24,17 @@ path_parsing:file:///Users/dev/Developer/flutter/.pub-cache/hosted/pub.dartlang.
 pedantic:file:///Users/dev/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/pedantic-1.8.0+1/lib/
 petitparser:file:///Users/dev/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/petitparser-2.4.0/lib/
 provider:file:///Users/dev/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/provider-4.0.4/lib/
-quiver:file:///Users/dev/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/quiver-2.0.5/lib/
+quiver:file:///Users/dev/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/quiver-2.1.3/lib/
 sky_engine:file:///Users/dev/Developer/flutter/bin/cache/pkg/sky_engine/lib/
-source_span:file:///Users/dev/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/source_span-1.5.5/lib/
+source_span:file:///Users/dev/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/source_span-1.7.0/lib/
 stack_trace:file:///Users/dev/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/stack_trace-1.9.3/lib/
 stream_channel:file:///Users/dev/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/stream_channel-2.0.0/lib/
 string_scanner:file:///Users/dev/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/string_scanner-1.0.5/lib/
 term_glyph:file:///Users/dev/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/term_glyph-1.1.0/lib/
-test_api:file:///Users/dev/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/test_api-0.2.11/lib/
+test_api:file:///Users/dev/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/test_api-0.2.15/lib/
 time_parser:file:///Users/dev/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/time_parser-1.1.0/lib/
 time_span:file:///Users/dev/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/time_span-1.0.3/lib/
 typed_data:file:///Users/dev/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/typed_data-1.1.6/lib/
 vector_math:file:///Users/dev/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/vector_math-2.0.8/lib/
-xml:file:///Users/dev/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/xml-3.5.0/lib/
+xml:file:///Users/dev/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/xml-3.6.1/lib/
 measure_your_life_app:lib/
```

Comment:
This file seems to be updated with each debug session - maybe we move that to gitignore?